examples/fox_in_a_hole/fox_in_a_hole.py

```diff
@@ -116,11 +180,21 @@ def print_history(self):
 
 
 class ClassicalGame(Game):
-    """Classical Fox-in-a-hole game."""
+    """Classical Fox-in-a-hole game.
+
+    In this version, we play the classical version of
+    Fox-in-a-Hole.
+
+    Each hole is either 0.0 (Fox not there) or 1.0 (Fox is there).
+
+    """
 
     def initialize_state(self):
-        self.state = self.hole_nr * [0.0]
-        index = self.rng.integers(low=0, high=self.hole_nr)
+        # All holes start as empty
+        self.state = self.number_of_holes * [0.0]
```

Comment:
maybe that's just me, but `[0.0] * self.number_of_holes` reads better, intention of making a list of numbers of a certain size is clearer

Reply:
Done.